backgroundScript.js

```diff
@@ -8,26 +8,20 @@ chrome.runtime.onInstalled.addListener(() => {
 
 chrome.tabs.onUpdated.addListener(
 	function (tabId, changeInfo, tab) {
-		if (tab.status === "complete") {
-			let url = tab.url.split('?')[0].split('/');
-			if (url[2] == 'github.com' && url[3] && url[4] && url[5] === 'pulls') {
-				chrome.tabs.sendMessage(tabId, {
-					message: 'githubUrl',
-					urls: tab.url,
-					repo_owner: url[3],
-					repo_name: url[4],
-					repo_function: url[5],
-				})
-			} else if (url[2] === "bitbucket.org" && url[5] === "pull-requests") {
-				chrome.tabs.sendMessage(tabId, {
-					message: 'bitbucketUrl',
-					urls: tab.url,
-				})
-			}
-			// for showing all tracked repo
-			else if (url[2] == 'github.com' && url[3]) {
-				// for woking on this url https://github.com/Alokit-Innovations type
-				if (url[3] && url[4] == undefined) {
+		chrome.storage.sync.get(["websiteUrl", "userId"]).then(({ websiteUrl, userId }) => {
+			if (tab.status === "complete") {
+				let url = tab.url.split('?')[0].split('/');
+				console.log("[addListener]", url);
+				if (url[2] == 'github.com' && url[3] && url[4] && url[5]==="pull" && url[6] && url[7] == 'files') {
+					chrome.tabs.sendMessage(tabId, {
+						message: 'showImpFileInPR',
+						urls: tab.url,
+						repo_owner: url[3],
+						repo_name: url[4],
+						repo_function: url[5],
+						userId: userId
+					})
+				} else if (url[2] == 'github.com' && url[3] && url[4] && url[5]) {
```

Comment:
What have you done here? It seems that you have removed the `githubUrl` message completely from this file. Please check if this chrome extension still highlights the relevant PRs in the `pulls` page.

Reply:
you already did the changes in background script in new pr, do I need to fix this ?